app/views/frontEndInputs.scala.html

```diff
@@ -5,4 +5,12 @@
 <input type="hidden" class="identity-provider-name" value="@frontEndInfo.identityProviderName">
 <input type="hidden" class="identity-pool-id" value="@frontEndInfo.identityPoolId">
 <input type="hidden" class="stage" value="@frontEndInfo.stage">
-<input type="hidden" class="region" value="@frontEndInfo.region">
\ No newline at end of file
+<input type="hidden" class="region" value="@frontEndInfo.region">
+
+@if(frontEndInfo.cognitoEndpointOverride.isDefined) {
+<input type="hidden" class="cognito-endpoint-override" value="@frontEndInfo.cognitoEndpointOverride">
```

Comment:
Indentation of line 11: `<input ... > ...`

Reply:
Fixed.